routers/api/v1/api.go

```diff
@@ -649,6 +649,10 @@ func Routes() *web.Route {
 
 		m.Group("/user", func() {
 			m.Get("", user.GetAuthenticatedUser)
+			m.Group("/preferences", func() {
```

Comment:
Why preferences here?

Reply:
since this are no repo settings or system settings but user specific settings ... can move it to `/user/settings` if you think it's better @zeripath